src/models/generated/DCLine.jl

```diff
@@ -0,0 +1,119 @@
+#=
+This file is auto-generated. Do not edit.
+=#
+
+#! format: off
+
+"""
+    mutable struct DCLine <: DCBranch
+        name::String
+        available::Bool
+        active_power_flow::Float64
+        arc::Arc
+        r::Float64
+        active_power_limits_from::MinMax
+        active_power_limits_to::MinMax
+        l::Float64
+        services::Vector{Service}
+        ext::Dict{String, Any}
+        internal::InfrastructureSystemsInternal
+    end
+
+a Standard DC line.
+
+# Arguments
+- `name::String`
+- `available::Bool`
+- `active_power_flow::Float64`
+- `arc::Arc`
+- `r::Float64`: Resistance system per-unit value
+- `active_power_limits_from::MinMax`
+- `active_power_limits_to::MinMax`
+- `l::Float64`: Inductance system per-unit value
+- `services::Vector{Service}`: Services that this device contributes to
+- `ext::Dict{String, Any}`
+- `internal::InfrastructureSystemsInternal`: power system internal reference, do not modify
+"""
+mutable struct DCLine <: DCBranch
```

Comment:
I don't think we need them both, the T model generalizes fine.

Reply:
The T-Model is way more advanced for many applications. Power Flows could suffice with a DC line with a resistor.

If we want to use the T-model for a Power Flow on which we only care about the series resistor, we would have to create a T-Line with a shunt = 0, and a resistor with value `0.5 r`, then we would need to aggregate both resistors.

If we include a shunt, then we will need to define this virtual bus similar to the three-winding transformer.

My suggestion is keeping the DCLine and use this one for initial implementations on PSI. In any, case we can use the T-model and ignore everything for PSI, if you think about this.

TLDR: Use DCLine for PSI studies, use T-model for PSID.